fulp_modules/ipc/code/surgery/restore_chassis_looks.dm

```diff
@@ -0,0 +1,44 @@
+/datum/surgery/robot_restore_looks
+	name = "Restore chassis looks"
+	steps = list(
+	/datum/surgery_step/weld_plating,
+	/datum/surgery_step/restore_paintjob)
+
+	requires_bodypart_type = BODYPART_ROBOTIC
+	possible_locs = list(BODY_ZONE_CHEST)
+	desc = "A procedure that welds the robotic limbs back into the patient's preferred state aswell as re-applying their paintjob."
+
+/datum/surgery_step/restore_paintjob
+	name = "spray paint"
+	implements = list(
+		/obj/item/toy/crayon/spraycan = 100)
+	time = 58
```

Comment:
let apcs do these surgeries on themselves

Reply:
or at least let the spray paint one be used on yourself; it really shouldn't even be a surgery, tbh